.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,36 +6,41 @@ labels: ''
 assignees: ''
 
 ---
-<!--Before filing a bug
+<!--
+Before filing a bug
 - Ensure the bug reproduces on the latest version of the app.
-- Search existing issues and make sure this issue is not already filed.-->
+- Search existing issues and make sure this issue is not already filed.
+-->
 
 **Describe the bug**
-<!--A clear and concise description of what the bug is.-->
+<!-- A clear and concise description of what the bug is. -->
 
 **Steps To Reproduce**
-<!--Steps to reproduce the behavior:
+<!--
+Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
-4. See error-->
+4. See error
+-->
 
 **Expected behavior**
-<!--A clear and concise description of what you expected to happen.-->
+<!-- A clear and concise description of what you expected to happen. -->
 
 **Screenshots**
-<!--If applicable, add screenshots to help explain your problem.-->
+<!-- If applicable, add screenshots to help explain your problem. -->
 
-**Device and Application Information (please complete the following information):**
+**Device and Application Information**
  - OS Build:
  - Architecture:
  - Application Version:
 
-<!--Run the following commands in Powershell and copy/paste the output.
+<!--
+Run the following commands in Powershell and copy/paste the output.
 " - OS Build: $([Environment]::OSVersion.Version)"
 " - Architecture: $((Get-AppxPackage -Name Microsoft.WindowsCalculator).Architecture)"
 " - Application Version: $((Get-AppxPackage -Name Microsoft.WindowsCalculator).Version)"
 -->
```

Comment:
Can we add an extra line to know if the user has the dev version installed too? (the version number would be better but not available):

 " - Dev Installed: $(!((Get-AppxPackage -Name Microsoft.WindowsCalculator.Dev) -eq $null))"

Reply:
Agree that versioning would be interesting, but how does knowing _just_ whether the dev version is installed help debug?

Reply:
It can be helpful to know:
- if the bug is related to the dev or the public version, the odds are very high that the contributor tried with the dev version if installed. It's useful when we have some troubles to reproduce the bug.
- if the contributor is also a developer and already compiled the application, in this case, we can ask him/her to test a PR in review, send a dump file, or put a breakpoint to know the value of a variable.

Reply:
Makes sense, will add both (good call IRT region).

> the odds are very high that the contributor tried with the dev version if installed

Perhaps it would be better to also ask whether bug repros in store/dev/both to help make this super clear?

Reply:
It will be great too! While we are speaking about that, in addition to the Start menu shortcut, should we also change the name of the app (Display Name), so "Calculator - Dev" will be displayed in the titlebar? (You already override the .appxmanifest, so it should not impact your build system)

_Hard to distinct the 2 versions_
![image](https://user-images.githubusercontent.com/1226538/55605728-aaf5f700-572a-11e9-9f1c-43991bee3156.png)

Reply:
> should we also change the name of the app (Display Name), so "Calculator - Dev"

I like it. Can you open up a new issue to track this?

Reply:
I will!